### PR TITLE
prefix the tag for builds

### PR DESCRIPTION
**Pull request by mrluc**
I think? Took approach from `generateId` in model helpers. Seems like it would exist as a top-level helper but don't know goland approach for this sort of thing.

### CODE REVIEW

**Comment by ddollar** [comment]
How about `convox-start-` as the prefix just so it's super clear where they came from.

File: api/manifest/manifest.go [hunk 1]
```diff
@@ -782,12 +782,16 @@ func runPrefix(prefix, executable string, args ...string) error {
 
 var randomAlphabet = []rune("abcdefghijklmnopqrstuvwxyz")
 
-func randomString(size int) string {
+func randomString(prefix string, size int) string {
 	b := make([]rune, size)
 	for i := range b {
-		b[i] = randomAlphabet[rand.Intn(len(randomAlphabet))]
+		idx, err := rand.Int(rand.Reader, big.NewInt(int64(len(randomAlphabet))))
```
Comment (ddollar) [inline api/manifest/manifest.go]: Let's leave the simpler version of the `randomAlphabet[]`selection in place. Did this version get copied from somewhere else?

**Comment by mrluc** [comment]
@ddollar made those changes

**Comment by ddollar** [comment]
## Release Playbook
- [x] Rebase against master
- [x] Pass checks
- [x] Release branch (20160226224447-incremental-concurrency)
- [x] Pass CI (https://circleci.com/gh/convox/rack/603)
- [x] Code review
- [x] Merge into master
- [x] Release master (20160226234213)
- [x] Pass CI (https://circleci.com/gh/convox/rack/604)
- [x] Update dev and testing racks
- [x] Publish release
- [x] Release CLI

**Comment by ddollar** [comment]
Thanks for the patch!

**Pull request merged**